ruby.rb

```diff
@@ -0,0 +1,10 @@
+class BadFlog
+  def some_method
+    super
+  rescue StandardError => error
+
+    if this(&wat=foo) == true
```

Comment:
Useless assignment to variable - wat.<br>Surrounding space missing for operator =.